src/handlers/expenditures/expenditureAdded.ts

```diff
@@ -85,5 +89,13 @@ export default async (event: ContractEvent): Promise<void> => {
       expenditureId: databaseId,
       fromDomainId: getDomainDatabaseId(colonyAddress, domainId),
     });
+
+    sendExpenditureUpdateNotifications({
+      colonyAddress,
+      creator: ownerAddress,
```

Comment:
This is an interesting one, for expenditures made using staking, this will be the extension's address. If you wanted to show the address of the user creating it, you'd need to send the notification on `ExpenditureMadeViaStake` event. However, you can't easily tell if it's a staked expenditure in this handler.

I think there are 2 options:
- fetch the next event log and check if it's `ExpenditureMadeViaStake`
- get current staked expenditure installation address and check if it matches owner address

Reply:
Oh very interesting... okay, I'll keep a note of this for when we tackle staked expenditures then, and I'll link this in that ticket. Thank you!